identify/extensions.py

```diff
@@ -73,6 +73,7 @@
     'idr': {'text', 'idris'},
     'inc': {'text', 'inc'},
     'ini': {'text', 'ini'},
+    'install': {'text', 'bash', 'alpm'},
```

Comment:
some googling shows other things using ".install" as a file extension, I'm not sure this is unambiguously arch packaging -- thoughts?

Reply:
Bummer. Seems some people use it for linux distributables in the widest sense. What might be enough is to factor in the executable bit. I checked and in the arch context this file does not have the executable bit set, but I can only assume that in a different context the bit is set.
If that is not good enough then I don't see a way around reading the file and determine the type from the function names used. In the arch context you have to define at least one function following the pattern `post_*`. Not sure how often you had to deal with this kind of ambiguity. It seems this case can't be solved by the current identify algorithm.

Reply:
usually in ambiguity we lean on the side of caution and don't include the extension 🤷 

there intentionally aren't complicated discovery rules (bits set, file contents, magic numbers, etc.) so it probably means `.install` isn't doable :(

Reply:
Too bad, but I get your point. Slightly off topic: do you know if pre-commit can be persuaded to feed this file type to the shellcheck hook for instance regardless of identify not classifying it as bash script?

Reply:
you can do:

```yaml
-   id: shellcheck
-   id: shellcheck
    types: [file]
    files: \.install$
```